ironic/drivers/modules/drac/raid.py

```diff
@@ -884,6 +972,65 @@ def _check_node_raid_jobs(self, task):
                 self._clear_raid_config_job_failure(node)
                 self._set_clean_failed(task, config_job)
 
+        if finished_job_ids and not node.driver_internal_info.get(
+                'raid_config_job_failure'):
+            if 'raid_config_substep' in node.driver_internal_info:
+                if node.driver_internal_info['raid_config_substep'] == \
+                        'delete_reset_config':
+                    controllers = list()
+                    no_jobs_required = False
+                    for controller_id in \
+                            node.driver_internal_info[
+                                'raid_config_parameters']:
+                        controller_cap = clear_foreign_config(node,
+                                                              controller_id)
+                        controller = {'raid_controller': controller_id,
+                                      'is_reboot_required': controller_cap[
+                                          'is_reboot_required']}
+                        controllers.append(controller)
+                        no_jobs_required = no_jobs_required == controller_cap[
+                            'is_commit_required']
```

Comment:
I think the above line should be:
jobs_required = jobs_required or controller_cap['is_commit_required']

Reply:
done